src/main/java/ua/goit/note/NoteController.java

```diff
@@ -0,0 +1,24 @@
+package ua.goit.note;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/note/list")
```

Comment:
The path above the class I think should be /note.
In the findAll method /list which will be added and then there will be /note/list

Reply:
When User Logs in it should be a view of all notes of the user hence /note/list 
When selecting specific note then it will note but with specific note id to it which is findById with relaton of user.